bigquery/deploy/README.md

```diff
@@ -61,9 +63,11 @@
 * Run Elasticsearch on GKE
   * Deploy Elasticsearch. From project root:
     ```
-    cd kubernetes-elasticsearch-cluster
+    cd kubernetes-elasticsearch-cluster <MY_DATASET>
```

Comment:
Move <MY_DATASET> to next line

nit remove angle brackets. Usually README's have MY_DATASET. (Angle brackets are used in localhost:9200/<MY_DATASET>_fields to make it stand out more.)

Reply:
Oops, done.